htdocs/install/mysql/migration/17.0.0-18.0.0.sql

```diff
@@ -83,6 +83,9 @@ ALTER TABLE llx_bank_account ADD COLUMN owner_country_id integer DEFAULT NULL;
 
 ALTER TABLE llx_prelevement_bons ADD COLUMN fk_bank_account integer DEFAULT NULL;
 
+ALTER TABLE llx_propal ADD COLUMN model_pdf_pos_sign VARCHAR(10) DEFAULT NULL AFTER last_main_doc;
```

Comment:
Can you provide an example of value you plzn to store in this field ?

Reply:
It's value 25:10:15:30

posX:posY:Height:Width